test/library/characterization/test_qubit_spectroscopy.py

```diff
@@ -281,3 +288,22 @@ def test_parallel_experiment(self):
 
         for child_data in par_data.child_data():
             self.assertRoundTripSerializable(child_data)
+
+    def test_circuit_roundtrip_serializable(self):
+        """Test circuits round trip JSON serialization"""
+        exp_helper = SpectroscopyHelper(
+            line_width=2e6,
+            iq_cluster_centers=[((-1.0, -1.0), (1.0, 1.0))],
+            iq_cluster_width=[0.2],
+        )
+        backend = MockIQBackend(
```

Comment:
Why not using one of the regular fake backends, e.g. FakeWashingtonV2? Probably for the circuits serialization test it should suffice

Reply:
You are right. I thought the `FakeBackend` couldn't produce this circuit.  I changed the backend to FakeWashingtonV2 as suggested.